BB-SC/MWLayer/Diagnostics/BB_Diagnostic_Policy_Manager.md

```diff
@@ -0,0 +1,77 @@
+
+# Diagnostic Policy Manager
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Diagnostics
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+MWLayer
+
+## Known Implementation
+
+## ID (unique name)
+
+## Description
+<!-- General Description of the BB -->
+Diagnostic service applications are interpreted as the diagnostic services according to ISO 14229. Define service at vehicle level to control different access level or different permission. Each level requires previous authentication, where authentication, can remain vehicle specific.
+
```

Comment:
I wonder if this type of capability shouldn't be framed to be much more generic/fundamental (than specifically applying for UDS)? And the capability would then also be used for implementing diagnostic access levels, but also any other authentication tasks?
Strongly recommend to have consider uProtocol for generic capabilities model and implementation work like this.

Reply:
Agreed. Needs discussion in WP2.

Reply:
generalize to "policy_manager" (change name)
add "expert_discussion" "in discussion"

Reply:
done